### ISSUE TITLE
EAS-1 External Command Seat has life support

### ISSUE THREAD

**Issue by MaximeBrean** [steamp0rt/Kerbalism#268]
With Kerbalism v2 and KSP 1.5.1, the external command seat feature all life support functionality.
It's probably due to the fact that the seat now has ```CrewCapacity = 1``` in stock game.

**Comment by Gordon-Dry** [comment]
@steamp0rt I guess just removing the MODULE[Habitat] from either the part seatExternalCmd or from all parts containing the MODULE[KerbalSeat] should be enough.

I suggest because modders just could add/edit new parts with that module, the 2nd solution should be used.

It's a simple change in System/Habitat.cfg:
```
@PART[*]:HAS[#CrewCapacity[>0],!MODULE[KerbalSeat]]:NEEDS[FeatureHabitat]:FOR[Kerbalism]
{
  MODULE
  {
	name = Habitat
  }
}
```
I will create a branch for that and PR it.

**Comment by Gordon-Dry** [comment]
If anybody provides a modded seat with kinda life support, it should be a closed system.

Further support of having an external command seat have life support should be provided by the mods devs themselves, like for example kinda hose that is connected to the suit with KIS/KAS.

**Comment by MaximeBrean** [comment]
Thanks for reactivity !
I just tested it in my setup and it only removed the "Habitat" module. The seat still have all "ProcessController" and reliability modules.

Apparently, all ProcessCrontroller modules (scrubber, air pump, ...) are configured into profiles config files (```Profiles/*.cfg```)
The reliability module is added from ```System/Reliability.cfg``` file.

So I added ```,!MODULE[KerbalSeat]``` in both ```System/Reliability.cfg``` (line 34) and ```Profiles/Default.cfg``` (line 503) and it's now regular stock seat :)
